reactDashboard/src/components/HeatMap/Heatmap.css

```diff
@@ -2,34 +2,58 @@
   overflow-y: auto;
   max-height: 600px;
   height: 100%;
-  background-color: #ffffff;
+  background-color: white;
+}
+
+.darkmode .heatmap {
+  background-color: rgb(64, 64, 64);
+  box-shadow: none;
+}
+
+.heatmap-container {
+  padding: 0;
 }
 
 .heatmap table {
-  overflow-y: auto;
+  border-collapse: separate;
+  border-spacing: 0;
 }
 
 .heatmap table th {
   vertical-align: top;
   position: sticky;
+  position: -webkit-sticky;
   top: 0;
-  z-index: 2;
   background-color: white;
   color: #133a53;
 }
 
-.heatmap table th .subheading {
+.darkmode .heatmap table th {
+  color: #f2f2f2;
+  background-color: rgb(64, 64, 64);
+}
+
+.heatmap table th div.subheading {
   font-weight: normal;
 }
 
+.table th {
+  border-top: none;
+}
+
+.darkmode .table thead th {
+  border-bottom: 1px solid #020202;
+}
+
 .heatmap table tr.area td {
-  border: 0;
-  padding: 0;
   vertical-align: middle;
+  border-top: 1px solid #f2f2f2;
 }
 
-.heatmap table tr.area:first-child td {
-  padding-bottom: 10px;
+.darkmode .heatmap table tr.area td {
+  color: #f2f2f2;
+  background-color: rgb(64, 64, 64);
+  border-top: 1px solid #020202;
```

Comment:
Only need to set border-top-color here

Reply:
refactored

Reply:
re-inserted - lost the lines when I took out "1 px solid"

Reply:
did you change the attribute to 'border-top-color' ?

Reply:
refactored